dali/operators/decoder/cache/cached_decoder_impl.cc

```diff
@@ -93,27 +93,42 @@ void CachedDecoderImpl::CacheStore(const std::string& file_name, const uint8_t *
 DALI_SCHEMA(CachedDecoderAttr)
   .DocStr(R"code(Attributes for cached decoder.)code")
   .AddOptionalArg("cache_size",
-      R"code(**`mixed` backend only** Total size of the decoder cache in megabytes. When provided, decoded
-images bigger than `cache_threshold` will be cached in GPU memory.)code",
+      R"code(Applies **only** to the ``mixed`` backend type.
+
+Total size of the decoder cache in megabytes. When provided, the decoded images
+that are larger than the ``cache_threshold`` value will be cached in GPU memory.
```

Comment:
```suggestion
that are larger than ``cache_threshold`` will be cached in GPU memory.
```
Maybe it's not perfect English, but it reads better, at least for programmers.
You can say "larger than 5". Replacing 5 with a variable name is very natural. In this - and similar - cases, I'd say that less is more.

Reply:
Done